
======= Running functional tests ======

python -m unittest discover -v 
Got exception when processing response to url=/TESTAPI/unitTests method=POST data={}
  Response status = 500
  Resonse headers: 
    content-length  =  14029
    connection  =  Keep-Alive
    x-request-id  =  2d807ebbaf27d3ed7b70c6521dab7596
    server  =  WEBrick/1.3.1 (Ruby/1.9.3/2012-12-25)
    x-runtime  =  6.081656
    date  =  Fri, 15 Feb 2013 17:17:14 GMT
    content-type  =  text/html; charset=utf-8
  Data string: <unknown
